### ISSUE TITLE
Support Bitcoin Gold / Bitcoin Platinum

### ISSUE THREAD

**Issue by wjcloud** [paritytech/parity-bitcoin#471]
Parity team would rule the world if parity-bitcoin supports Bitcoin Gold and Bitcoin Platinum.

About Bitcoin Gold: Bitcoin Gold is a hardfork of bitcoin with a change of pow algorithm to equihash - same algorithm as Zcash. It has implemented SIGHASH_FORKID features used from Bitcoin Cash in order to protect 2-way replay attack. The homepage is here: http://btcgpu.org. Twitter: https://twitter.com/bitcoingold

You can refer the source of BTCGPU: https://github.com/BTCGPU/BTCGPU

About Bitcoin Platinum: Bitcoin Platinum is a hardfork of bitcoin with a change of pow algorithm to equihash - same algorithm as Zcash and added SEGWIT2X features made by btc1 project. It has implemented SIGHASH_FORKID features used from Bitcoin Cash in order to protect 2-way replay attack.
and removed premine features from Bitcoin Gold. The homepage is here: http://btcplt.org(Currently under maintenance. Twitter: https://twitter.com/BTCPLT 

You can refer the source of BitcoinPlatinum: https://github.com/BitcoinPlatinum/BitcoinPlatinum.

**Comment by wjcloud** [comment]
Plus) Some of the source codes are not publicly opened unless they release the main-net.
When they open the main-net, we could start the project!

**Comment by debris** [comment]
I don't think that support for yet another bitcoin fork is anywhere on our roadmap. Personally, I also see no reason to support it, but if someone makes a non-intrusive pr, I will consider merging it.
